### ISSUE TITLE
Client disconnect with Purplicious Cow Head & Cool T-Shirt

### ISSUE THREAD

**Issue by LemADEC** [capnkirok/Inventory-Pets#275]
As of 1.12-1.5.3, player gets disconnected from the server when holding the Purplicious Cow Head:
```
[17:34:34] [Server thread/WARN] [minecraft/NetworkSystem]: Failed to handle packet for /82.46.164.239:37205
net.minecraft.util.ReportedException: Ticking player
>       at net.minecraft.entity.player.EntityPlayerMP.onUpdateEntity(EntityPlayerMP.java:459) ~[oq.class:?]
>       at net.minecraft.network.NetHandlerPlayServer.redirect$onPlayerTick$zin000(NetHandlerPlayServer.java:2047) ~[pa.class:?]
>       at net.minecraft.network.NetHandlerPlayServer.update(NetHandlerPlayServer.java:173) ~[pa.class:?]
>       at net.minecraftforge.fml.common.network.handshake.NetworkDispatcher$1.update(NetworkDispatcher.java:209) ~[NetworkDispatcher$1.class:?]
>       at net.minecraft.network.NetworkManager.processReceivedPackets(NetworkManager.java:285) ~[gw.class:?]
>       at net.minecraft.network.NetworkSystem.networkTick(NetworkSystem.java:180) [oz.class:?]
>       at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:790) [MinecraftServer.class:?]
>       at net.minecraft.server.dedicated.DedicatedServer.updateTimeLightAndEntities(DedicatedServer.java:396) [nz.class:?]
>       at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668) [MinecraftServer.class:?]
>       at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
>       at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]
Caused by: java.lang.NoSuchMethodError: net.minecraft.entity.player.InventoryPlayer.func_70440_f(I)Lnet/minecraft/item/ItemStack;
>       at com.inventorypets.items.patreonHead.onUpdate(patreonHead.java:53) ~[patreonHead.class:?]
>       at net.minecraft.item.ItemStack.updateAnimation(ItemStack.java:526) ~[aip.class:?]
>       at net.minecraft.entity.player.InventoryPlayer.decrementAnimations(InventoryPlayer.java:363) ~[aec.class:?]
>       at net.minecraft.entity.player.EntityPlayer.onLivingUpdate(EntityPlayer.java:511) ~[aed.class:?]
>       at net.minecraft.entity.EntityLivingBase.onUpdate(EntityLivingBase.java:2170) ~[vp.class:?]
>       at net.minecraft.entity.player.EntityPlayer.onUpdate(EntityPlayer.java:234) ~[aed.class:?]
>       at net.minecraft.entity.player.EntityPlayerMP.onUpdateEntity(EntityPlayerMP.java:382) ~[oq.class:?]
>       ... 10 more
```

**Comment by capnkirok** [comment]
Thanks. Definitely a bug.

**Comment by LemADEC** [comment]
Cool T-Shirt also causes player to disconnect in a similar way:
```
17:55:50] [Server thread/WARN] [net.minecraft.network.NetworkSystem]: Failed to handle packet for /82.46.164.239:51464
net.minecraft.util.ReportedException: Ticking player
        at net.minecraft.entity.player.EntityPlayerMP.onUpdateEntity(EntityPlayerMP.java:459) ~[oq.class:?]
        at net.minecraft.network.NetHandlerPlayServer.redirect$onPlayerTick$zin000(NetHandlerPlayServer.java:2047) ~[pa.class:?]
        at net.minecraft.network.NetHandlerPlayServer.update(NetHandlerPlayServer.java:173) ~[pa.class:?]
        at net.minecraftforge.fml.common.network.handshake.NetworkDispatcher$1.update(NetworkDispatcher.java:209) ~[NetworkDispatcher$1.class:?]
        at net.minecraft.network.NetworkManager.processReceivedPackets(NetworkManager.java:285) ~[gw.class:?]
        at net.minecraft.network.NetworkSystem.networkTick(NetworkSystem.java:180) [oz.class:?]
        at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:790) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.updateTimeLightAndEntities(DedicatedServer.java:396) [nz.class:?]
        at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]
Caused by: java.lang.NoSuchMethodError: net.minecraft.entity.player.InventoryPlayer.func_70440_f(I)Lnet/minecraft/item/ItemStack;
        at com.inventorypets.items.patreonShirt.onUpdate(patreonShirt.java:53) ~[patreonShirt.class:?]
        at net.minecraft.item.ItemStack.updateAnimation(ItemStack.java:526) ~[aip.class:?]
        at net.minecraft.entity.player.InventoryPlayer.decrementAnimations(InventoryPlayer.java:363) ~[aec.class:?]
        at net.minecraft.entity.player.EntityPlayer.onLivingUpdate(EntityPlayer.java:511) ~[aed.class:?]
        at net.minecraft.entity.EntityLivingBase.onUpdate(EntityLivingBase.java:2170) ~[vp.class:?]
        at net.minecraft.entity.player.EntityPlayer.onUpdate(EntityPlayer.java:234) ~[aed.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.onUpdateEntity(EntityPlayerMP.java:382) ~[oq.class:?]
        ... 10 more
```

**Comment by capnkirok** [comment]
From this report, it looks like I have some client only code running on the Server. Rookie move, Cheers.

**Comment by capnkirok** [comment]
Ok, this is fixed in the upcoming 1.5.3.1 release
